Docs/source/Overview/software-overview.rst

```diff
@@ -0,0 +1,37 @@
+*****************
+Software Overview
+*****************
+
+The software is responsible for
+
+- Making sure that the Driver and Pit Crew can communicate
+- Collecting Data from peripherals and other systems
+- Sending data to the Pit Crew to determine status of vehicle
+
+Hardware Abstraction Layer (HAL)
+================================
+
+The `HAL <https://www.st.com/resource/en/user_manual/dm00105879-description-of-stm32f4-hal-and-ll-drivers-stmicroelectronics.pdf>`_ 
```

Comment:
I'm not sure if it's elsewhere in the docs, but it might be good to have something about using STMCube to generate the HAL configuration.

Reply:
Included link.